x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -35,7 +37,12 @@ interface TraceItem {
   impact: number;
 }
 
-const TRACE_COLUMNS = [
+interface Props {
+  items: ImpactItem[];
+  noItemsMessage: any;
+}
+
+const traceListColumns: ITableColumn[] = [
```

Comment:
The "I" prefix still seems pretty old-school to me, but I think it's the most pragmatic solution to add it, to avoid conflicts between component names and interfaces. Should we make it a rule to have it for all interfaces?

There is a ts-lint rule: https://palantir.github.io/tslint/rules/interface-name/

Reply:
I'm in favor of this, but perhaps during a general "TypeScript cleanup" phase post DT?

Reply:
Sure!